appliances/MinIO/appliance.sh

```diff
@@ -191,9 +194,34 @@ service_configure()
         chown -R minio-user:minio-user /opt/minio
     fi
 
-    # Edit MinIO environment variable file
-    if (( local_drives_count > 1 )); then
+
+    # If Multi-Node deployment, edit MinIO volumes
+    if [[ ${ONEAPP_MINIO_MULTI} =~ ^(yes|YES)$ ]]; then
+        local_subdomain=$(awk -F. '{ print $1 }' <<<${ONEAPP_MINIO_HOSTNAME})
+        local_domain=$(sed -e "s/[^.]*//" <<<"$ONEAPP_MINIO_HOSTNAME")
+        # Dynamically create hosts entries on /etc/hosts, getting IP for each node from onegate
+        # Do this before updating env file, so MinIO can connect to other nodes
+        # Get array of nodes IDs from MinIO role on service
+        local_nodes=($(onegate service show --json | jq -r '.SERVICE.roles[] | select(.name=="MinIO").nodes[].deploy_id'))
```

Comment:
How about using something like this stuff below instead? :thinking: 

```
onegate service show -j --extended | jq -r '.SERVICE.roles[]|select(.name=="MinIO").nodes[].vm_info.VM.TEMPLATE.CONTEXT.ETH0_IP'
```

or

```
onegate service show -j --extended | jq -r '.SERVICE.roles[]|select(.name=="MinIO").nodes[].vm_info.VM.TEMPLATE.NIC.IP'
```

You get the same IPs with a single API call, the if you want to do a loop:

```
onegate service show -j --extended | jq -r '.SERVICE.roles[]|select(.name=="MinIO").nodes[].vm_info.VM.TEMPLATE.NIC.IP' | while read IP
; do echo "IP -> $IP"; done
```